### PR TITLE
Add dropbox support to export data to cloud functionality

### PR DESCRIPTION
**Pull request by rosogon**
Change-type: minor

### CODE REVIEW

**Comment by rosogon** [comment]
@Exulansis : Could you review? I tested both owncloud and dropbox and they work.

File: src/clouds/dropbox.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Atos - initial API and implementation
+ ******************************************************************************/
+const dropbox = require('dropbox')
+const d = require('debug-levels')('agile-data-dropbox')
+require('isomorphic-fetch');
```
Comment (rosogon) [inline src/clouds/dropbox.js:15]: @Exulansis : Dropbox needs fetch in globals. Not sure if here is the best place for this require.

**Comment by Exulansis** [comment]
Will review this shortly, thank you for the PR

**Comment by rosogon** [comment]
Hi, @Exulansis . When do you expect to review the PR? Thanks

**Pull request merged**